dashboard/app/controllers/plc/enrollment_evaluations_controller.rb

```diff
@@ -19,7 +19,7 @@ def confirm_assignments
 
     modules_to_enroll_in = Plc::LearningModule.find([params[:content_module], params[:practice_module]])
 
-    if modules_to_enroll_in.size == 2 && modules_to_enroll_in.pluck(:module_type).sort == [Plc::LearningModule::CONTENT_MODULE, Plc::LearningModule::PRACTICE_MODULE]
+    if modules_to_enroll_in.pluck(:module_type).sort == [Plc::LearningModule::CONTENT_MODULE, Plc::LearningModule::PRACTICE_MODULE]
```

Comment:
The second part of the conditional implicitly checks that there are two elements, and I can't see any performance reason to shortcut this check. (We're running the query a couple of lines above either way.)

Also, TIL [`find` can be used to retrieve multiple objects.](https://guides.rubyonrails.org/active_record_querying.html#find)

Reply:
Oh, wow. I would have assumed that `.find([a, b])` is closer to `.find(a) || .find(b)` than `[.find(a), .find(b)]`